helpers/validator.js

```diff
@@ -0,0 +1,127 @@
+const steem = require('@steemit/steem-js');
+const validator = require('validator');
+const badDomains = require('../bad-domains');
+
+const accountNotExist = (rule, value, callback) => {
```

Comment:
Let's call this something else than callback to avoid confusion since it is not using the `(error, result) =>` pattern

Reply:
this is a funciton used by the react form for validation (i did not write it for this PR, it's old code); i'll leave a comment noting when/where it's used